src/js/shield_defs.js

```diff
@@ -2845,6 +2845,12 @@ export function loadShields(shieldImages) {
     Color.shields.white
   );
 
+  // Iraq
+  shields["IQ:national"] = roundedRectShield(
+    Color.shields.green,
+    Color.shields.white
```

Comment:
```suggestion
  shields["IQ:national"] = roundedRectShield(
    Color.shields.green,
    Color.shields.white,
    Color.shields.white,
    34
```

The shield images on [Wikimedia Commons](https://commons.wikimedia.org/wiki/Category:Diagrams_of_route_signs_of_Iraq) appear to be fixed-width.